README.md

```diff
@@ -40,7 +40,7 @@ Using acme-dns is a three-step process (provided you already have the self-hoste
 
 The method returns a new unique subdomain and credentials needed to update your record.
 Fulldomain is where you can point your own `_acme-challenge` subdomain CNAME record to.
-With the credentials, you can update the TXT response in the service to match the challenge token, later referred as \_\_\_validation\_token\_recieved\_from\_the\_ca\_\_\_, given out by the Certificate Authority.
+With the credentials, you can update the TXT response in the service to match the challenge token, later referred as \_\_\_validation\_token\_received\_from\_the\_ca\_\_\_, given out by the Certificate Authority.
```

Comment:
Fun fact: https://github.com/joohoi/acme-dns/pull/66#issuecomment-379536205

Reply:
You aren't alone: https://github.com/ossec/ossec-docs/pull/252/commits/1b447f032aceab026b57c4761ed115e4f6bb6b56 (and following)